homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayService.java

```diff
@@ -0,0 +1,72 @@
+package com.tutrit.java.quickstart.homework.lookarrayservice;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LookupArrayService {
+    static Logger log = LoggerFactory.getLogger("Array");
+
+    public int findMax(int[] array) {
+        if (array == null) return 0;
+       int arrayMax = array[0];
+        for (int j : array) {
+            if (arrayMax < j) {
+                arrayMax = j;
+            }
+        }
+        log.info("arrayMax  = {}", arrayMax);
+        return arrayMax;
+    }
+
+    public int findMin(int[] array) {
+        if (array == null) return 0;
+       int arrayMin = array[0];
+        for (int j : array) {
+            if (arrayMin > j) {
+                arrayMin = j;
+            }
+        }
+        log.info("arrayMin = {}", arrayMin);
+        return arrayMin;
+
+    }
+
+    public int indexOfMax(int[] array) {
+        int indexMax = -1;
+        if (array == null) return -1;
```

Comment:
That's ok, but as you've knew exceptions already I suggest that you take a look at lamaconda's enterprise approach for this check.

Reply:
fixed.